pkg/go/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openfga/language/pkg/go
 
-go 1.21
+go 1.21.3
```

Comment:
I think `go1.21.7` is the only one that would quite down the alerts.

Alternatively you can force it in the `main-go` workflow file to use go>=1.21.7

Reply:
I tried to set just `go 1.21` here, but it seems the newer Go tooling is a bit more strict around the value set here and (I think) it's the minimum from the list of dependencies.

I was leaning towards running lint/vulncheck with a newer version of Go so that we don't pickup irrelevant vuln warnings and then run tests against 1.21 and 1.22 to verify our supported range like in JS. What do you think?